numerapi/numerapi.py

```diff
@@ -1103,3 +1103,19 @@ def daily_submissions_performances(self, username: str) -> List[Dict]:
         performances = [p for p in performances
                         if any([p['correlation'], p['fnc'], p['mmc']])]
         return performances
+
+    def get_dataset_list(self):
```

Comment:
I didn't know about listDatasets, quite useful!

Please document the method like all the other methods  in the repository.

Reply:
yep we just added this. it shows you all the individual files that you can download